From eaa70a289288db1e8d90a2a9bcde435a8975fe74 Mon Sep 17 00:00:00 2001
From: Sergey Lukashevich <sergey.lukashevi4@gmail.com>
Date: Fri, 26 Jul 2019 21:58:34 +0300
Subject: [PATCH] Broken build on Windows: "http_parser.h(66,18): error C2039: 
 'runtime_error': is not a member of 'std'"

Add missing <stdexcept> include.

[Retrieved from:
https://github.com/darrenjs/wampcc/commit/eaa70a289288db1e8d90a2a9bcde435a8975fe74]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 include/wampcc/http_parser.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/wampcc/http_parser.h b/include/wampcc/http_parser.h
index a9405dd..4c20417 100644
--- a/include/wampcc/http_parser.h
+++ b/include/wampcc/http_parser.h
@@ -11,6 +11,7 @@
 #include <map>
 #include <string>
 #include <memory>
+#include <stdexcept>
 
 // types brought in from nodejs http-parser project
 struct http_parser;
